libs/librrgraph/src/io/rr_graph_writer.h

```diff
@@ -0,0 +1,32 @@
+/*
+ * This function writes the RR_graph generated by VPR into a file in XML format
+ * Information included in the file includes rr nodes, rr switches, the grid, block info, node indices
+ */
+
+#ifndef RR_GRAPH_WRITER_H
+#define RR_GRAPH_WRITER_H
+
+#include "rr_node.h"
+#include "rr_graph_type.h"
+#include "rr_graph_builder.h"
+#include "rr_graph_view.h"
+#include "rr_graph_fwd.h"
```

Comment:
You do not need to include the ``rr_graph_fwd.h``. It is already included in the ``rr_graph_builder.h`` and ``rr_graph_view.h``